quinn-proto/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "quinn-proto"
 version = "0.11.4"
 edition = "2021"
-rust-version = "1.67"
+rust-version = "1.70"
```

Comment:
Maybe move to workspace `Cargo.toml`?

Reply:
Is that a thing? Would be neat if so.

Reply:
Yes. See https://github.com/quinn-rs/quinn/pull/1940.